### PR TITLE
Delete functionality

### PR DESCRIPTION
**Pull request by dustin-oz**

### CODE REVIEW

File: app/javascript/components/pages/MyListing.js [hunk 1]
```diff
@@ -1,9 +1,47 @@
...
+import {NavLink} from 'react-router-dom'
 
 export default class MyListing extends Component {
   render() {
+    const {myBusiness} = this.props
     return (
-      <h1>MyListing</h1>
-    )
+      <>
+        <h3>My Business Listing</h3>
```
Comment (elysemontano) [inline app/javascript/components/pages/MyListing.js:10]: This seems like it should have been part of your protected index card.

File: app/javascript/components/pages/MyListing.js [hunk 2]
```diff
@@ -1,9 +1,47 @@
...
+        {myBusiness &&
+        <Card>
+          <Row className="cards">
+            {myBusiness?.map((business) => {
+              console.log(business.id);
```
Comment (elysemontano) [inline app/javascript/components/pages/MyListing.js:15]: Please remove this console log before merging

**Review by elysemontano — APPROVED** [review]
Please remove your console log, also please consider the trello card stories and make sure they match.  It seems like there is some work that might be more toward the protected index card.

File: app/javascript/components/pages/MyListing.js (hunk 2 above)
Comment (dustin-oz) [reply app/javascript/components/pages/MyListing.js]: Done

**Pull request merged**